cclib/parser/nwchemparser.py

```diff
@@ -1077,10 +1095,146 @@ def extract(self, inputfile, line):
                 while line.strip():
                     tokens = line.split()
                     assert len(tokens) == 8
-                    atomcoords_step.append([float(c) for c in tokens[2:5]])
+                    atomcoords_step.append([utils.float(c) for c in tokens[2:5]])
                     line = next(inputfile)
                 self.atomcoords.append(atomcoords_step)
 
+        # Extract Thermochemistry in au (Hartree)
+        #
+        # have to deal with :
+        # Temperature                      =   298.15K
+        # frequency scaling parameter      =   1.0000
+        # Zero-Point correction to Energy  =  259.352 kcal/mol  (  0.413304 au)
+        # Thermal correction to Energy     =  275.666 kcal/mol  (  0.439302 au)
+        # Thermal correction to Enthalpy   =  276.258 kcal/mol  (  0.440246 au)
+        # Total Entropy                    =  176.764 cal/mol-K
+        # - Translational                =   44.169 cal/mol-K (mol. weight = 448.1245)
+        # - Rotational                   =   37.018 cal/mol-K (symmetry #  =        1)
+        # - Vibrational                  =   95.577 cal/mol-K
+        # Cv (constant volume heat capacity) =  103.675 cal/mol-K
+        # - Translational                  =    2.979 cal/mol-K
+        # - Rotational                     =    2.979 cal/mol-K
+        # - Vibrational                    =   97.716 cal/mol-K
+        if line[1:12] == "Temperature":
+            self.set_attribute('temperature', utils.float(line.split()[2][:-1]))
+        if line[1:28] == "frequency scaling parameter":
+            self.set_attribute('pressure', utils.float(line.split()[4]))
+        if line[1:31] == "Thermal correction to Enthalpy" and hasattr(self, 'scfenergies'):
+            self.set_attribute('enthalpy', utils.float(line.split()[8]) + utils.convertor(self.scfenergies[-1], "eV", "hartree"))
+        if line[1:32] == "Zero-Point correction to Energy" and hasattr(self, 'scfenergies'):
+            self.set_attribute('zpve', utils.float(line.split()[8]))
+        if line[1:29] == "Thermal correction to Energy" and hasattr(self, 'scfenergies'):
+            self.set_attribute('electronic_thermal_energy', utils.float(line.split()[8]) + utils.convertor(self.scfenergies[-1], "eV", "hartree"))
+        if line[1:14] == "Total Entropy":
+            self.set_attribute('entropy', utils.convertor(1e-3 * utils.float(line.split()[3]),"kcal/mol","hartree"))
+
+        # extract vibrational frequencies (in cm-1)
+        if line.strip() == "Normal Eigenvalue ||           Projected Infra Red Intensities":
+            if not hasattr(self, 'vibfreqs'):

```

Comment:
Rather than this and the `self.vibfreqs.append` later, prefer `self.append_attribute("vibfreqs", ...)`.  Same goes for `vibirs`.